test/cppsuite/tests/search_near_02.cpp

```diff
@@ -173,15 +173,7 @@ class search_near_02 : public test_harness::test {
 
             auto &cursor_prefix = cursors[coll.id];
 
-            /*
-             * Pick a random timestamp between the oldest and now. Get rid of the last 32 bits as
-             * they represent an increment for uniqueness.
-             */
-            wt_timestamp_t ts = random_generator::instance().generate_integer(
-              (tc->tsm->get_oldest_ts() >> 32), (tc->tsm->get_next_ts() >> 32));
-            /* Put back the timestamp in the correct format. */
-            ts <<= 32;
-
+            wt_timestamp_t ts = tc->tsm->get_random_ts();
```

Comment:
Do you need to get rid of the uniqueness as before?

Reply:
Luckily not anymore, previously we needed to because get_next_ts() would increment the last half for uniqueness. Now since we can use get_time_now_s(), we don't need to care about uniqueness anymore.